### PR TITLE
Feature/fsm 112

### PR DESCRIPTION
**Pull request by mahendra-egov**

### CODE REVIEW

File: packages/modules/fsm/src/components/inbox/search.js [hunk 1]
```diff
@@ -1,95 +1,95 @@
...
+              <div
+                className="complaint-header"
+                style={{
+                  display: "flex",
+                  justifyContent: "space-between",
+                  width: "100%",
+                  marginBottom: "20px",
+                }}
+              >
+                <h2>SEARCH BY:</h2>
```
Comment (saurabh-egov) [inline packages/modules/fsm/src/components/inbox/search.js:61]: add localization keys

File: packages/modules/fsm/src/pages/MarkForDisposal.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      setPopup(false);
+    }
+  };
+
+  const handlePopupClose = () => {
+    setPopup(false);
+    setSelectedComponent(null);
+  };
+
+  const applications = React.useMemo(
```
Comment (saurabh-egov) [inline packages/modules/fsm/src/pages/MarkForDisposal.js:44]: Add todo

File: packages/modules/fsm/src/pages/MarkForDisposal.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      if (selectedApplicationNumbers.includes(application["Application No."])) {
+        totalWaste += application["Waste Collected"];
+      }
+    });
+    setTotalWasteCollected(totalWaste);
+  }, [selectedApplicationNumbers]);
+
+  const handleSubmit = () => {
+    const data = applications.filter((application) => selectedApplicationNumbers.includes(application["Application No."]));
+    console.log("%c MarkForDisposal: handleSubmit -> data ", "font-size:16px;background-color:#d0601a;color:white;", data);
```
Comment (saurabh-egov) [inline packages/modules/fsm/src/pages/MarkForDisposal.js:90]: hide console logs

File: packages/modules/fsm/src/pages/employee/ApplicationDetails.js [hunk 4]
```diff
@@ -181,6 +183,18 @@ const ApplicationDetails = (props) => {
     ],
   };
 
+  if (DSO) {
```
Comment (saurabh-egov) [inline packages/modules/fsm/src/pages/employee/ApplicationDetails.js:186]: add todo

File: packages/modules/fsm/src/pages/employee/Inbox.js [hunk 5]
```diff
@@ -28,16 +29,17 @@ const Inbox = () => {
...
-    return <MobileInbox data={applications} isLoading={isLoading || isIdle} onFilterChange={handleFilterChange} onSearch={onSearch} />;
-  } else {
-    return (
-      <div>
-        <Header>{t("ES_COMMON_INBOX")}</Header>
-        <DesktopInbox data={applications} isLoading={isLoading || isIdle} onFilterChange={handleFilterChange} onSearch={onSearch} />
-      </div>
-    );
+  if (applications?.length !== null) {
+    if (DSO || isMobile) {
```
Comment (saurabh-egov) [inline packages/modules/fsm/src/pages/employee/Inbox.js:33]: check this

File: packages/css/src/pages/employee/index.scss [hunk 6]
```diff
@@ -96,10 +96,27 @@
...
   }
 }
+
+.multi-select-container {
+  .disposal-text {
+    @apply font-bold;
+  }
+
+  .disposal-info {
+    margin-top: "16px";
```
Comment (saurabh-egov) [inline packages/css/src/pages/employee/index.scss:114]: can we use tailwind

File: packages/modules/fsm/src/components/inbox/search.js (hunk 1 above)
Comment (mahendra-egov) [reply packages/modules/fsm/src/components/inbox/search.js:61]: done

File: packages/modules/fsm/src/pages/MarkForDisposal.js (hunk 2 above)
Comment (mahendra-egov) [reply packages/modules/fsm/src/pages/MarkForDisposal.js:44]: done

File: packages/modules/fsm/src/pages/MarkForDisposal.js (hunk 3 above)
Comment (mahendra-egov) [reply packages/modules/fsm/src/pages/MarkForDisposal.js:90]: done

File: packages/modules/fsm/src/pages/employee/ApplicationDetails.js (hunk 4 above)
Comment (mahendra-egov) [reply packages/modules/fsm/src/pages/employee/ApplicationDetails.js:186]: done

File: packages/modules/fsm/src/pages/employee/Inbox.js (hunk 5 above)
Comment (mahendra-egov) [reply packages/modules/fsm/src/pages/employee/Inbox.js:33]: done

File: packages/css/src/pages/employee/index.scss (hunk 6 above)
Comment (mahendra-egov) [reply packages/css/src/pages/employee/index.scss:114]: yes, we can. done.

**Pull request merged**